experiments/compositions/composition/expanders.mk

```diff
@@ -0,0 +1,56 @@
+###### ----------- Build protos --------------------------------------------
+
+##@ Protos
+
+PROTOC ?= /usr/bin/protoc
+.PHONY: protoc
+protoc: $(PROTOC) ## Download protoc if necessary
+$(PROTOC):
+	sudo apt install -y protobuf-compiler
+	protoc --version
+	go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+	go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
+
+.PHONY: build-protos
+build-protos: protoc
+	$(PROTOC) --go_out=. --go_opt=paths=source_relative --go-grpc_out=. --go-grpc_opt=paths=source_relative proto/expander.proto
+
+###### ----------- Expander client command---------------------------
+
+##@ Expander expand client command
+
+.PHONY: build-expand
+build-expand: build-protos fmt vet ## Build binary.
+	go build -v -o expanders/bin/expand ./expanders/expand
+
+clean-expand: ## clean binary.
+	rm -fr expanders/bin/expand
+
+###### ----------- Jinja2 Expander ----------------------------
+
+##@ Jinja2 expander pod
+
+JINJA2_IMG ?= $(IMG_REGISTRY)/expander-gjinja2:$(IMG_VERSION)
```

Comment:
Personally prefer variable declaration at the top

Reply:
moved it to the top.